media/css/firefox/new/trailhead/thanks.scss

```diff
@@ -11,6 +11,76 @@ $image-path: '/media/protocol/img';
 @import '../../../../protocol/css/components/picto-card';
 @import '../../../../protocol/css/templates/card-layout';
 
+// Chrome post-download instructions experiment.
```

Comment:
Should we split this CSS out into its own file? The selectors added here are quite broad, but whilst they don't interfere with the regular page they are not needed for the non-experimental version?

Reply:
While the non-experimental version doesn't need the styles, the new variant does. I separated the styles into it's own file and added both to it's bundle. :+1: